common/src/commonMain/kotlin/org/hisp/dhis/common/screens/Components.kt

```diff
@@ -1,67 +1,21 @@
 package org.hisp.dhis.common.screens
 
 enum class Components(val label: String) {
-    ICON_BUTTON("Icon Button"),
-    BUTTON("Button"),
-    FORM_SHELLS("Form Shells"),
-    INPUT_TEXT("Input Text"),
-    INPUT_LONG_TEXT("Input Long Text"),
-    INPUT_POSITIVE_INTEGER_OR_ZERO("Input Positive Integer Or Zero"),
-    INPUT_NEGATIVE_INTEGER("Input Negative Integer"),
-    INPUT_PERCENTAGE("Input Percentage"),
-    INPUT_POSITIVE_INTEGER("Input Positive Integer"),
-    INPUT_INTEGER("Input Integer"),
-    INPUT_NUMBER("Input Number"),
-    INPUT_LETTER("Input Letter"),
-    BOTTOM_SHEET_HEADER("Bottom Sheet Header"),
-    BOTTOM_SHEET("Bottom Sheet "),
-    TAGS("Tags"),
-    SECTIONS("Sections"),
-    RADIO("Radio"),
-    CHECK_BOX("Check Box"),
-    PROGRESS("Progress"),
-    SUPPORTING_TEXT("Supporting Text"),
-    LEGEND("Legend"),
-    LEGEND_DESCRIPTION("Legend description"),
-    BUTTON_BLOCK("Button block"),
-    CHIPS("Chips"),
-    BADGES("Badges"),
-    METADATA_AVATAR("Metadata Avatar"),
-    INPUT_RADIO_BUTTON("Input Radio Button"),
-    SWITCH("Switch"),
-    INPUT_MATRIX("Input Matrix"),
-    INPUT_SEQUENTIAL("Input Sequential"),
-    INPUT_QR_CODE("Input QR code"),
-    INPUT_CHECK_BOX("Input Check Box"),
-    BARCODE_BLOCK("Barcode Block"),
-    INPUT_AGE("Age Field"),
-    INPUT_YES_ONLY_SWITCH("Input yes only switch"),
-    INPUT_YES_ONLY_CHECKBOX("Input yes only checkbox"),
-    INPUT_YES_NO_FIELD("Input Yes/No field"),
-    LIST_CARD("List Card"),
-    CARD_DETAIL("Card Detail"),
-    INPUT_PHONE_NUMBER("Input Phone Number"),
-    INPUT_LINK("Input Link"),
-    INPUT_EMAIL("Input Email"),
-    INPUT_BARCODE("Input Barcode"),
-    CAROUSEL_BUTTONS("Carousel buttons"),
-    INPUT_POLYGON("Input Polygon"),
-    INPUT_ORG_UNIT("Input Org. Unit"),
-    IMAGE_BLOCK("Image Block"),
-    INPUT_FILE_RESOURCE("Input File Resource"),
-    INPUT_DROPDOWN("Input Dropdown"),
-    INPUT_DATE_TIME("Input Date Time"),
-    INPUT_COORDINATE("Input Coordinate"),
-    INPUT_SIGNATURE("Input Signature"),
-    INPUT_UNIT_INTERVAL("Input Unit Interval"),
-    INPUT_IMAGE("Input Image"),
-    SEARCH_BAR("Search bar"),
-    INPUT_NOT_SUPPORTED("Input Not Supported"),
-    FULL_SCREEN_IMAGE("Full Screen Image"),
-    ORG_TREE_BOTTOM_SHEET("Org Tree Bottom Sheet"),
-    INDICATOR("Indicator Input"),
+    ACTION_INPUTS("Action inputs"), // 14
```

Comment:
are these commented numbers needed? if so, could you add an explanation in the comment?

Reply:
nice catch! they are not needed no, I will remove them